ebaysdk/inventorymanagement/__init__.py

```diff
@@ -0,0 +1,259 @@
+        config_file   -- YAML defaults (default: ebay.yaml)
+        debug         -- debugging enabled (default: False)
+        warnings      -- warnings enabled (default: False)
+        uri           -- API endpoint uri (default: /services/search/FindingService/v1)
+        token         -- eBay application/user token
+        version       -- version number (default: 1.0.0)
+        https         -- execute of https (default: False)
+        proxy_host    -- proxy hostname
+        proxy_port    -- proxy port number
+        timeout       -- HTTP request timeout (default: 20)
+        parallel      -- ebaysdk parallel object
+        response_encoding -- API encoding (default: XML)
+        request_encoding  -- API encoding (default: XML)
+        """
+
+        super(Connection, self).__init__(method='POST', **kwargs)
+
+        self.config=Config(domain=kwargs.get('domain', 'api.ebay.com'),
+                           connection_kwargs=kwargs,
+                           config_file=kwargs.get('config_file', 'ebay.yaml'))
+
+        # override yaml defaults with args sent to the constructor
+        self.config.set('domain', kwargs.get('domain', 'api.ebay.com'))
+        self.config.set('uri', '/selling/inventory/v1')
+        self.config.set('https', True)
+        self.config.set('warnings', True)
+        self.config.set('errors', True)
+        self.config.set('siteid', None)
+        self.config.set('response_encoding', 'XML')
+        self.config.set('request_encoding', 'XML')
+        self.config.set('proxy_host', None)
+        self.config.set('proxy_port', None)
+        self.config.set('token', None)
+        self.config.set('iaf_token', None)
+        self.config.set('appid', None)
+        self.config.set('version', '1.0.0')
+        self.config.set('service', 'InventoryManagement')
+        self.config.set('doc_url', 'http://developer.ebay.com/Devzone/store-pickup/InventoryManagement/index.html')
+
+        self.datetime_nodes = ['starttimefrom', 'timestamp', 'starttime',
+                               'endtime']
+        self.base_list_nodes = [
+        ]
+
+    endpoints = {
+        'addinventorylocation': 'locations/delta/add',
+        'addinventory': 'inventory/delta/add',
+        'deleteinventory': 'inventory/delta/delete',
+        'deleteinventorylocation': 'locations/delta/delete',
+    }
+
+    def build_request_url(self, verb):
+        url = super(Connection, self).build_request_url(verb)
+        endpoint = self.endpoints[verb.lower()]
+        return "{0}/{1}".format(url, endpoint)
+
+    def build_request_headers(self, verb):
+        return {
+            "Authorization": "TOKEN {0}".format(self.config.get('token')),
+            "Content-Type": "application/xml"
+        }
+
+    def build_request_data(self, verb, data, verb_attrs):
+        xml = "<?xml version=\"1.0\" encoding=\"utf-8\"?>"
+        xml += "<" + verb + "Request>"
+        xml += dict2xml(data)
```

Comment:
However I was stupid, I should copy it from Trading that I fixed because I ran again on same problems with utf-8 today morning. Will be fixed tomorrow!